CONTRIBUTING.md

```diff
@@ -0,0 +1,170 @@
+
+### Update release config
+
+1. Open a PR from the `dev-*` branch of a particular configuration to the equivalent `release-*` branch
+2. QA checks will run to ensure the configuration meets criteria for a released configuration, and to ensure consistency of released configurations.
+3. Checks will also run to test if changes break reproducibility with the current major version config tag on the target branch. For example, if you are opening a PR on the `release-1deg_jra55_iaf` branch, and the last tagged version on this branch is `release-1deg_jra55_iaf-1.2`, the checksums between the config in your PR and the checksum in the config tag are compared.
+4. A comment will be posted on the PR when this is completed, notifying you whether the checksums match (in this example meaning a minor bump to `*-1.3`), or are different (meaning a major bump to `*-2.0`).
+5. Optionally, you can now modify your PR and get more reproducibility checks. Particularly in the case where bitwise reproducibility should be retained this is an opportunity to modify the configuration to enable this.
+6. Bump the version using the `!bump [major|minor]` command depending on the result of the reproducibility check. Additionally, if the checksums are different, the updated checksum will be automatically committed to the PR. Bumping the version in some way is a requirement before the PR will be mergable.
+7. Merge the PR
+
+### Common Changes Required
+
+#### Required metadata
+
+Each configuration should have a metadata file, `metadata.yaml`, which uses the ACCESS-NRI [experiment schema](https://github.com/ACCESS-NRI/schema/blob/main/au.org.access-nri/model/output/experiment-metadata/README.md). Note that required fields of the schema are removed from file validations checks as experiment specific fields such as UUIDs should not be set for released configurations. The following fields must be set in `metadata.yaml`:
+
+##### version
+
+Use the existing `release-*` version. If there isn't an existing version set to `null`.
+
+##### realm
+
+Acceptable values for this field must be set as a sequence:
+
+```yaml
+realm:
+    - <acceptable value 1>
+    - <acceptable value 2>
+    # - ...
+```
+
+##### nominal resolution
+
+Choose the appropriate value for the resolution used:
+
+| Config resolution | Nominal Resolution |
+| -- | -- |
+| 1&deg; | 100 km |
+
+`<Add more as applicable>`
+
+These are sourced from [the CMIP6 controlled vocabulary](https://github.com/WCRP-CMIP/CMIP6_CVs/blob/main/CMIP6_nominal_resolution.json). If your resolution differs from those listed you will need to make a pull request to add it to this documentation and the QA checks.
+
+##### keywords
+
+We have a "controlled vocabulary of keywords to prevent a proliferation of synonyms that mean the same thing, and to make it easy to populate these fields:
+
+| Topic | Keywords (mutually exclusive) |
+|--|--|
+| `<topic>` | `<keywords>` |
```

Comment:
```suggestion
| Spatial extent | `global`|
| Model | `access-esm1.5` |
```

I am sure there are other keywords we could suggest, but I'm not familiar enough with ESM to know. Will add to the ESM stand-up to ask.